src/rdpc/analysesProcessor.ts

```diff
@@ -0,0 +1,281 @@
+import fetch from "node-fetch";
+import {
+  Run,
+  RunsByAnalysesByDonors,
+  Analysis,
+  DonorRunStateMap,
+  RunsByInputAnalyses,
+} from "./types";
+import _ from "lodash";
+import logger from "logger";
+import promiseRetry from "promise-retry";
+import HashCode from "ts-hashcode";
+
+const buildQuery = (studyId: string, from: number, size: number): string => {
+  const query = `
+  fragment AnalysisData on Analysis {
+    analysisId
+    analysisType
+    studyId
+    donors {
+      donorId
+    }
+  }
+
+  query {
+    SequencingExperimentAnalyses: analyses(
+      filter: {
+        analysisType: "sequencing_experiment"
+        studyId: "${studyId}"
+      },
+      page: {from: ${from}, size: ${size}}
+    ) {
+      ...AnalysisData
+      runs: inputForRuns {
+        runId
+        state
+        sessionId
+        repository
+        inputAnalyses {
+          analysisId
+        }
```

Comment:
I don't think we're using all of this info eh? let's take out the unused stuff and update the types :)